pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -188,8 +186,8 @@ func (r *InferenceServiceReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		result, err := reconciler.Reconcile(isvc)
 		if err != nil {
 			r.Log.Error(err, "Failed to reconcile", "reconciler", reflect.ValueOf(reconciler), "Name", isvc.Name)
-			r.Recorder.Eventf(isvc, v1.EventTypeWarning, "InternalError", err.Error())
-			r.updateStatus(isvc, deploymentMode)
+			r.Recorder.Event(isvc, v1.EventTypeWarning, "InternalError", err.Error())
+			_ = r.updateStatus(isvc, deploymentMode)
```

Comment:
I stumbled upon this line and was wondering why this update call exists. The error was not handled, so it would be good to mark it explicit to be ignored. I am not sure if this is the right behaviour though.

Reply:
This is handled in PR #3378